sample/src/main/res/layout/activity_main.xml

```diff
@@ -688,6 +685,71 @@
                 app:textSize="20sp" />
 
         </com.addisonelliott.segmentedbutton.SegmentedButtonGroup>
+
+        <com.addisonelliott.segmentedbutton.SegmentedButtonGroup
+            android:id="@+id/buttonGroup_vectorDrawable"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_margin="4dp"
```

Comment:
Oh yeah, the margin bottom will need to be 60dp here.

This was just so I could move the button to be up higher on the device screen for some reason.